src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private Random random = new Random();
+
+    public Figure getRandomFigure() {
+
+        Figure randomFigure = null;
+
+        int randomType = random.nextInt(5 - 1 + 1) + 1;
+        switch (randomType) {
+            case 1 -> randomFigure = new Circle();
+            case 2 -> randomFigure = new Rectangle();
+            case 3 -> randomFigure = new Square();
+            case 4 -> randomFigure = new IsoscelesTrapezoid();
+            case 5 -> randomFigure = new RightTriangle();
+            default -> getDefaultFigure();
```

Comment:
init value of your figures here, not inside

Reply:
"not inside" where ? Inside each figure class ?